src/server/routes/index.ts

```diff
@@ -15,6 +16,9 @@ router.use(core);
 // request reset password routes
 router.use(noCache, reset);
 
+// request sign in routes
+router.use(noCache, signIn);
```

Comment:
another small thing, but not for this PR, I feel like `noCache` is used for most of our routes, so maybe we want to opt into routes that we would actually like to cache.

Reply:
Yeah, so make not using a cache the default. This makes sense